### ISSUE TITLE
failing test in gcs_repo_test.go

### ISSUE THREAD

**Issue by michelleN** [kubernetes/helm#488]
```
--- FAIL: TestListCharts (0.30s)
	gcs_repo_test.go:69: expected one chart in list, got 2
```
`make test` fails on master. It's now returning 2 charts instead of 1. 
I believe this is because we're calling out to a real gcs bucket called kubernetes-charts-testing which now has in real life 2 charts (http://storage.googleapis.com/kubernetes-charts-testing). 

... We probably shouldn't be calling out to a real bucket in a test unless it's an integration test, and regardless of what kind of test, we might want to think about putting in a cleanup step.

**Comment by jackgr** [comment]
Already fixed in #486, which is awaiting review. See also #468, #469, #470 re: not using a real bucket.

**Comment by jackgr** [comment]
Fixed. by #486.
